mlos_bench/README.md

```diff
@@ -47,152 +54,118 @@ With a [JSON5](https://spec.json5.org) [config file](./mlos_bench/config/) and c
 
-6. From here onwards we assume we are in the project root directory.
-Ensure you have a conda environment (`mlos`) set up for executing `mlos_bench`.
-Create and activate the environment with:
+For database like SQLite or DuckDB, there is no need for an additional config file.
+The data will be stored locally in a file, e.g., `./mlos_bench.duckdb`.
+See [`sqlite.jsonc`](./mlos_bench/config/storage/sqlite.jsonc) or [`duckdb.jsonc`](./mlos_bench/config/storage/duckdb.jsonc) for more details.
 
-    ```sh
-    conda env create -f conda-envs/mlos.yml
-    conda activate mlos
-    ```
+### 5. Create a top-level configuration file for your MLOS setup
 
-7. Connect the [`Environment`](./mlos_bench/environments/), [`Service`](./mlos_bench/services/), [`Storage`](./mlos_bench/storage/), and [`Optimizer`](./mlos_bench/optimizers/) configurations in a top-level [JSON5](https://spec.json5.org) configuration - say, `config_bench.jsonc`:
+We provide a few examples of such files in [`./mlos_bench/config/cli/`](./mlos_bench/config/cli).
+For example, [`azure-redis-opt.jsonc`](./mlos_bench/config/cli/azure-redis-opt.jsonc) is a configuration for optimizing Redis VM on Azure and saving the results in a local SQLite database.
+Likewise, [`azure-redis-bench.jsonc`](./mlos_bench/config/cli/azure-redis-bench.jsonc) is a setup to run a single Redis benchmark (and, again, save the results in SQLite).
 
-    ```jsonc
-    {
-        "config_path": [                                 // Locations of config files and scripts
-            "my-config",
-            "mlos_bench/examples"
-        ],
+CLI configs like those are meant to connect several MLOS components together, namely:
 
-        "environment": "env-azure-ubuntu-redis.jsonc",   // Root config (location relative to config_path)
+- Benchmarking environment (configured in [`environments/root/root-azure-redis.jsonc`](./mlos_bench/config/environments/root/root-azure-redis.jsonc));
+- Optimization engine ([`optimizers/mlos_core_flaml.jsonc`](./mlos_bench/config/optimizers/mlos_core_flaml.jsonc));
+- Storage for experimental data ([`storage/sqlite.jsonc`](./mlos_bench/config/storage/sqlite.jsonc));
 
-        "tunable_values": [                              // Key/value pairs of tunable parameters. Uses config_path
-            "tunable-values-example.json"
-        ],
+They also refer to other configs, e.g.
 
-        "globals": [                                     // Config generated at step 2. Uses config_path
-            "global_config.json"
-        ],
+- Reusable config snippets in `"config_path"` section, and
+- Additional config files containing sensitive data like DB passwords and Azure credentials.
 
-        "teardown": false,                               // Do not shutdown/deprovision a VM
+> Make sure that the files `./global_config_azure.json` and `./global_config_storage.json` you created in steps 3 and 4 are included in the `"globals"` section of your CLI config.
```

Comment:
```suggestion
> Make sure that the files `./global_config_azure.json` and `./global_config_storage.json` you created in the previous steps are included in the `"globals"` section of your CLI config.
```

Just a maintenance tweak in case we renumber things.